nailgun/entity_mixins.py

```diff
@@ -82,34 +82,31 @@ def raise_task_timeout():
     timer = threading.Timer(timeout, raise_task_timeout)
 
     # Poll until the task finishes. The timeout prevents an infinite loop.
+    path = '{0}/foreman_tasks/api/tasks/{1}'.format(server_config.url, task_id)
     try:
         timer.start()
-        path = '{0}/foreman_tasks/api/tasks/{1}'.format(
-            server_config.url,
-            task_id
-        )
```

Comment:
I've moved this out of the `try` block because I see no reason for it to be inside.